### DISCUSSION TITLE
Docker version of 2024.1 comes with antispam filter

### DISCUSSION THREAD

**Discussion by dhdeangelis** [INSPIRE-MIF/helpdesk-validator#1079] [Q&A] [answered]
In docker version of 2024.1 the antispam filter is present. This is not optimal for a local instance.

![image](https://github.com/INSPIRE-MIF/helpdesk-validator/assets/51515911/1a7190c0-4e85-4d51-b2ac-19125796166a)

**Comment by DanielMartinPerezdeLeon** [comment, marked as answer]
Dear @dhdeangelis,

thank you very much for reaching out and communicate this issue.

To deactivate the spam filter, you can access the file (.../inspire-validator-2024.1/ui.zip/validator/js/config.js) inside the release zip and set the property captchaEnabled to "false".

Best regards,

**Comment by dhdeangelis** [comment]
> To deactivate the spam filter, you can access the file (.../inspire-validator-2024.1/ui.zip/validator/js/config.js) inside the release zip and set the property captchaEnabled to "false".

Thank you @DanielMartinPerezdeLeon that solved the issue.
